packages/devtools_app/lib/src/screens/memory/panes/leaks/leaks_pane.dart

```diff
@@ -273,6 +274,35 @@ class _LeaksPaneState extends State<LeaksPane> {
   }
 }
 
+class _LeaksHelpLink extends StatelessWidget {
+  const _LeaksHelpLink({Key? key}) : super(key: key);
+
+  static const _documentationTopic = 'leaks';
+
+  @override
+  Widget build(BuildContext context) {
+    return HelpButtonWithDialog(
+      gaScreen: analytics_constants.memory,
+      gaSelection:
+          analytics_constants.topicDocumentationButton(_documentationTopic),
+      dialogTitle: 'Memory Leak Detection Help',
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.end,
+        children: [
+          const Text('The memory leak detection tab helps to detect\n'
+              'and troubleshoot some types of memory leaks.'),
```

Comment:
What types? I'd prefer to change "some types" so that it's less ambiguous.

```suggestion
          const Text('Use the memory leak detection tab to detect\n'
              'and troubleshoot some types of memory leaks.'),
```